topcoffea/json/test_samples/UL17_private_ttH_for_CI.json

```diff
@@ -2,7 +2,7 @@
   "xsec": 0.2151,
   "year": "2017",
   "treeName": "Events",
-  "histAxisName": "ttHJet_privateUL16APV",
+  "histAxisName": "ttHJet_privateUL17",
```

Comment:
I'm a bit curious about what's going on here, did you determine that this cfg was mislabeled (i.e. the root file was already a UL17 sample, so you just adjusted the name to match), or did you actually swap out the root file?